ament_cmake_export_linker_flags/cmake/ament_cmake_export_linker_flags-extras.cmake.in

```diff
@@ -0,0 +1,10 @@
+# generated from ament_cmake_export_linker_flags/cmake/ament_cmake_export_linker_flags-extras.cmake.in
```

Comment:
This file shouldn't be  called `-extras` since that suffix commonly only refers to the file above.